requirements.txt

```diff
@@ -4,6 +4,6 @@ jsonschema>=4.6,<5.0
 libcomps>=0.1.20.post1,<0.2
 productmd~=1.33.0
 pulpcore>=3.17,<3.25
-PyGObject>=3.40,<3.46
+PyGObject>=3.40,<3.45
```

Comment:
Just to confirm there never has been a 3.45.* (or they all got deleted). Then this _is_ the right thing to do.

https://pypi.org/project/PyGObject/#history